extensions/modules/widget-utils/notes/feature-hit_test.md

```diff
@@ -0,0 +1,4 @@
+# Path hit testing
+
+* A function `hitTestPath` has been added providing hit testing of a
```

Comment:
@peter-b: The problem with geometric approaches is numerical accuracy. Path rendering is done using linear step-wise approximation so kinda side-step and tricky corner edge cases which end up with things blowing up to infinity.